### PR TITLE
Feature/check cc and bcc fields

### LINKED ISSUE #4: Assertions for Bcc
Check the Bcc field like he To & From

### PR DESCRIPTION
**Pull request by jimmypuckett**
Added assertions for Cc, Bcc & Reply To. Closes #4, #5 & #8

### CODE REVIEW

File: src/MailTracking.php [hunk 1]
```diff
@@ -90,6 +87,38 @@ public function recordMail(Swift_Message $email)
     }
 
     /**
+     * Assert that the last email was bcc'ed to the given address.
+     *
+     * @param string        $bcc
+     * @param Swift_Message $message
```
Comment (ssfinney) [inline src/MailTracking.php]: Do you want to use that, or the convention `@param Swift_Message|null $message`?

**Comment by ssfinney** [comment]
Coveralls never reported the test coverage. Can you manually trigger another commit to see if it will report?

File: src/MailTracking.php [hunk 2]
```diff
@@ -90,6 +87,38 @@ public function recordMail(Swift_Message $email)
...
+     *
+     * @return PHPUnit_Framework_TestCase $this
+     */
+    protected function seeEmailCc($cc, Swift_Message $message = null)
+    {
+        $this->assertArrayHasKey($cc, (array)$this->getEmail($message)
+                                                  ->getCc(), "No email was cc'ed to $cc.");
+
+        return $this;
+    }
```
Comment (ssfinney) [inline src/MailTracking.php:119]: Why not allow someone to pass in an array of emails for the CC and BCC tests?

File: src/MailTracking.php (hunk 1 above)
Comment (jimmypuckett) [inline src/MailTracking.php]: I will make that change. Thanks.

File: src/MailTracking.php (hunk 2 above)
Comment (jimmypuckett) [inline src/MailTracking.php:119]: I made a separate issue for that #18 so I want to do that in a different feature.

File: src/MailTracking.php (hunk 1 above)
Comment (jimmypuckett) [inline src/MailTracking.php]: I have made that change, so this PR is ready to review again.

**Pull request merged**